website/project/model.py

```diff
@@ -880,6 +875,11 @@ def pk(self):
         return self._id
 
     @property
+    def logs(self):
+        """ List of logs associated with this node"""
+        return NodeLog.find(Q('node', 'eq', self._id)).sort('date')
```

Comment:
Should these be sorted on date descending?

As is `node.logs[0]` will be the node creation log and `node.logs[-1]` will be the latest. Is that the current behavior/desired?